x-pack/plugins/ml/public/application/components/data_grid/feature_importance/use_classification_path_data.tsx

```diff
@@ -165,9 +199,87 @@ export const buildClassificationDecisionPathData = ({
     }
     return undefined;
   });
+
+  // get the baseline for the current class from the trained_models metadata
+  const baselineClass = baselines.find(
+    (bl) => getStringBasedClassName(bl.class_name) === getStringBasedClassName(currentClass)
+  );
   const filteredFeatureImportance = mappedFeatureImportance.filter(
     (f) => f !== undefined
   ) as ExtendedFeatureImportance[];
 
-  return buildDecisionPathData(filteredFeatureImportance);
+  const finalResult: DecisionPathPlotData = filteredFeatureImportance
+    // sort by absolute importance so it goes from bottom (baseline) to top
+    .sort(
+      (a: ExtendedFeatureImportance, b: ExtendedFeatureImportance) =>
+        b.absImportance! - a.absImportance!
+    )
+    .map((d) => [d[FEATURE_NAME] as string, d[FEATURE_IMPORTANCE] as number, NaN]);
```

Comment:
qq: why do you need a `NaN` here? Just to comply `DecisionPathPlotData` type to provide it to the chart?

Reply:
That's correct. The NaN will be overriden later with actual numeric values.

Reply:
Got it, please check this comment https://github.com/elastic/kibana/pull/80229/files#r507704700